lint-staged.config.js

```diff
@@ -11,5 +11,7 @@ module.exports = {
     // For that reason, we move the `--onlyChanged` flag next to it.
     'npm run lint:js -- --reporters=jest-silent-reporter --onlyChanged',
     'flow focus-check',
+    'npm run build:todos',
+    'git add TODOS.md',
```

Comment:
Adding is something you, I think, want to do in a lint-staged script cause there you know if you actually have to.

Reply:
Ahh got it. I thought this was some sort of hook that got run on every file change, individually. But it actually just runs once?

Reply:
Yup. The staged files are passed as an argument but we dont do anything with them so this should be fine. Give it a try. Otherwise I misread the lint-staged docs.

Reply:
ahhh makes total sense. Thanks! 😍